routing/turns_generator.cpp

```diff
@@ -201,10 +201,10 @@ bool DiscardTurnByHighwayClass(TurnCandidates const & possibleTurns, TurnInfo co
 /// * - or any alternative turn to a similar road if the turn's angle is less than kMaxAbsAngleSameRoadClass degrees (wider than SlightTurn)
 /// * - or any alternative turn to a smaller road if it's GoStraight or SlightTurn.
 /// * Returns true otherwise.
-/// \param possibleTurns is all possible ways out from a junction.
+/// \param candidates_without_uturn is all possible ways out from a junction except uturn.
 /// \param turnInfo is information about ingoing and outgoing segments of the route.
 /// it is supposed that current turn is GoStraight or SlightTurn
-bool DiscardTurnByNoAlignedAlternatives(TurnCandidates const & possibleTurns, TurnInfo const & turnInfo,
+bool DiscardTurnByNoAlignedAlternatives(std::vector<TurnCandidate> const & candidates_without_uturn, TurnInfo const & turnInfo,
```

Comment:
```suggestion
bool DiscardTurnByNoAlignedAlternatives(std::vector<TurnCandidate> const & candidatesWithoutUturn, TurnInfo const & turnInfo,
```
Надо соблюдать наше текущее соглашение о форматировании.